exporter/datadogexporter/internal/clientutil/retrier.go

```diff
@@ -80,8 +80,8 @@ func (r *Retrier) DoWithRetries(ctx context.Context, fn func(context.Context) er
 		}
 
 		backoffDelayStr := backoffDelay.String()
-		r.logger.Info(
-			"Request failed with retriable errors. Will retry the request after interval.",
+		r.logger.Debug(
```

Comment:
Is there a scenario where we drop the data ? For such a case we want a error or warn log

Reply:
Yes, if the request eventually fails and data is dropped, the eventual error is returned to the exporterhelper and is logged as an error: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/internal/clientutil/retrier.go#L78